docs/openapi/components/schemas/credentials/BillOfLadingCredential.yml

```diff
@@ -294,6 +318,166 @@ example: |-
+              },
+              "fulfillmentCenter": {
+                "type": [
+                  "Organization"
+                ],
+                "name": "Florida Contact Org",
+                "email": "Florida91@example.net",
+                "phoneNumber": "+1-555-104-1126",
+                "location": {
+                  "type": [
+                    "Place"
+                  ],
+                  "address": {
+                    "type": [
+                      "PostalAddress"
+                    ],
+                    "name": "Hahn LLC",
+                    "streetAddress": "786 Pfeffer Plains",
+                    "addressLocality": "West Ottilie",
+                    "addressRegion": "Nebraska",
+                    "postalCode": "50878-0870",
+                    "addressCountry": "Cook Islands"
+                  }
+                }
+              },
+              "orderedItem": {
+                "type": [
+                  "Product"
+                ],
+                "manufacturer": {
+                  "type": [
+                    "Organization"
+                  ],
+                  "name": "Florida Contact Org",
+                  "email": "Florida91@example.net",
+                  "phoneNumber": "+1-555-104-1126",
+                  "location": {
+                    "type": [
+                      "Place"
+                    ],
+                    "address": {
+                      "type": [
+                        "PostalAddress"
+                      ],
+                      "name": "Hahn LLC",
+                      "streetAddress": "786 Pfeffer Plains",
+                      "addressLocality": "West Ottilie",
+                      "addressRegion": "Nebraska",
+                      "postalCode": "50878-0870",
+                      "addressCountry": "Cook Islands"
+                    }
+                  }
+                },
+                "sizeOrAmount": {
+                  "type": [
+                    "QuantitativeValue"
+                  ],
+                  "unitCode": "1",
+                  "value": "1"
+                },
+                "weight": {
+                  "type": [
+                    "QuantitativeValue"
+                  ],
+                  "unitCode": "1",
+                  "value": "1"
+                },
+                "depth": {
+                  "type": [
+                    "QuantitativeValue"
+                  ],
+                  "unitCode": "1",
+                  "value": "1"
+                },
+                "width": {
+                  "type": [
+                    "QuantitativeValue"
+                  ],
+                  "unitCode": "1",
+                  "value": "1"
+                },
+                "height": {
+                  "type": [
+                    "QuantitativeValue"
+                  ],
+                  "unitCode": "1",
+                  "value": "1"
+                },
+                "productPrice": {
```

Comment:
This object is empty. I suggest removing it - product price is not relevant on a BL.